### DISCUSSION TITLE
（灌水）你们永恒邦布使我下降那一关大概是用多少时间完成的

### DISCUSSION THREAD

**Discussion by stormcat-efforia** [share121/inter-knot#3339] [其他]
在网上经常看到有人吐槽那一关极度费时间，弄得我现在都不敢打

**Comment by clcl-maker** [comment]
还没打到QAQ

**Comment by 502awa** [comment]
别听他们的很快的<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" ><img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" ><img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Reply by stormcat-efforia** [reply]
直觉告诉我不简单

**Comment by hbxjvsnsvhxg** [comment]
我也不知道啊我也没打

**Comment by 3165341164** [comment]
只能说一群人计较蚊子腿的奖励，打到十五层就能拿满菲林了<img src="https://bbs-static.miyoushe.com/static/2023/04/28/b71f9bae78003f98a47f0d6e0765324d_7897172668253324041.png" alt="米仔_休息" width="75" >
